docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -610,62 +665,82 @@ Configurations, Producer Properties and Advanced Producer Configurations.
 ====== Consumer Properties
 
 .Consumer configurable properties of spring-cloud-azure-stream-binder-servicebus
-[cols="<,<,<", options="header"]
+[cols="<,<,<,<", options="header"]
 |===
-|Property | Type |Description
+|Property | Type |Default |Description
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.requeue-rejected
 |boolean
+|false
 |If the failed messages are routed to the DLQ.
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.checkpoint-mode
 | CheckpointMode
+| RECORD
 | The checkpoint mode of checkpointing message. The supported modes are `MANUAL` and `RECORD`.
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.max-concurrent-calls
 | Integer
+| 1
 | Max concurrent messages that the Service Bus processor client should process.
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.max-concurrent-sessions
 |Integer
+|1
 | Maximum number of concurrent sessions to process at any given time.
 
-|*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.session-aware
+|*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.concurrency
+|Integer
+|1
+|if sessionsEnabled is true, controls the maximum number of concurrent sessions to process, else controls the max concurrent calls of service bus message handler and the size of fixed thread pool that handles user's business logic.
+
+Deprecated, replaced with maxConcurrentSessions when sessionsEnabled is true and maxConcurrentCalls when sessionsEnabled is false.
+
+|*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.session-enabled
 | Boolean
+| false
 | Whether session is enabled.
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.prefetch-count
 | Integer
+| 1
 | The prefetch count of the Service Bus processor client.
 
 |*spring.cloud.stream.servicebus.bindings.<binding-name>.consumer*.sub-queue
 | SubQueue
+| SubQueue.NONE
```

Comment:
Let's keep the default value format of all Enum properties in consistency, just use the string format. So here we can modify it from "SubQueue.NONE" to "none".